src/components/common-components/modal/error-modal/error.modal.list.item.component.tsx

```diff
@@ -10,10 +10,21 @@ interface Options {
 }
 
 export default function ModalErrorListItem({ error }: Options): JSX.Element {
+  const displayTitle = error.title && error.title !== "Nie rozpoznano błędu";
   return (
     <div className="error-list-item">
       <div className="red-rectangle" />
-      <div className="error-text">{error.message}</div>
+      <div className="error-modal">
+        {displayTitle && (
+          <div className="error-title">
+            <p className="error-title-content">{error.title}</p>
+          </div>
+        )}
```

Comment:
Jeżeli błąd zawiera tytuł, to myślę, że sensownie byłoby go wyświetlić. Te, które nie mają są wyświetlane tak jak wcześniej.
![tytuł](https://user-images.githubusercontent.com/48785655/112359346-77bf2080-8cd1-11eb-932c-4210c19f84aa.png)
![brak tytułu](https://user-images.githubusercontent.com/48785655/112359360-7b52a780-8cd1-11eb-96fc-d48aad691103.png)

Reply:
to samo co wyżej, pierwszy dzien miesiaca - pierwsza kolumna